app/components/bulk_upload_error_row_component.html.erb

```diff
@@ -18,16 +20,50 @@
         <% end %>
 
         <%= table.with_body do |body| %>
-          <% bulk_upload_errors.each do |error| %>
+          <% bulk_upload_errors.reject { |error| error.category == "soft_validation" }.each do |error| %>
             <% body.with_row do |row| %>
-              <% row.with_cell(header: true, text: error.cell) %>
-              <% row.with_cell(text: question_for_field(error.field)) %>
-              <% row.with_cell(text: error.error) %>
-              <% row.with_cell(text: error.field.humanize) %>
+                <% row.with_cell(text: error.cell) %>
+                <% row.with_cell(text: question_for_field(error.field)) %>
+                <% row.with_cell(header: true, text: error.error) %>
```

Comment:
I don't think this semantically makes sense as a header, we probably want to keep the header as is and instead apply different styling to this row (probably strong tag)